engine/app/components/citizens_advice_components/input.rb

```diff
@@ -10,7 +10,7 @@ def initialize(name:, label:, type:, options: nil)
       @label = label
       @type = fetch_or_fallback(
         allowed_values: allowed_type_values,
-        given_value: type.to_sym,
+        given_value: type.present? ? type.to_sym : nil,
```

Comment:
Might be able to get away with `given_value.presense` here. Should return the symbol or nil in this case.